### PR TITLE
Jest

### PR DESCRIPTION
**Pull request by justinr1234**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: packages/ripple-keypairs/.eslintrc.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
     browser: true, // Enable browser global variables
     node: true, // Enable node global variables & Node.js scoping
     es2020: true, // Add all ECMAScript 2020 globals and automatically set the ecmaVersion parser option to ES2020
-    mocha: true, // Add Mocha testing global variables
+    jest: true, // Add Mocha testing global variables
```
Comment (JST5000) [inline packages/ripple-keypairs/.eslintrc.js:22]:
```suggestion
    jest: true, // Add Jest testing global variables
```

File: packages/ripple-address-codec/jest.config.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
+// Jest configuration for api
+const base = require('../../jest.config.base.js')
+
 module.exports = {
-  "roots": [
```
Comment (mvadari) [inline packages/ripple-address-codec/jest.config.js:2]: IMO if it ain't broke don't fix it

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -35,7 +36,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```
Comment (mvadari) [inline .github/workflows/nodejs.yml:39]: For now, I would keep the node versions the same - they can be adjusted in another PR.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -169,3 +169,14 @@ We have a low-traffic mailing list for announcements of new `xrpl.js` releases.
 If you're using the XRP Ledger in production, you should run a [rippled server](https://github.com/ripple/rippled) and subscribe to the ripple-server mailing list as well.
 
 + [Subscribe to ripple-server](https://groups.google.com/g/ripple-server)
+
+# How to upgrade gatsby & npm packages
```
Comment (mvadari) [inline CONTRIBUTING.md:173]: Does this repo use gatsby? I thought that was a web framework
